app/services/resource_recommendations.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
+CRISIS_PREVENTION_TAGS = %w[
```

Comment:
Since this is being used across all locale, we might to expand this list of tags to cater for the other locales!

Reply:
Maybe we could abstract it into its own yaml with multiple translations.

Reply:
Ah yes these have been translated to our other locales! But I could move these translation keys into it's own yaml. I think I'll revisit that in another PR if we end up adding a lot more. Thanks for bringing this up!